### PR TITLE
Basic support for portrait orientation, closes #1

### LINKED ISSUE #1: Portrait orientation
Currently the Big FT display is fairly hopeless in portrait orientation.  We need some media queries to make this work better.  Basic layout is probably (excuse the hasty artwork):

image

As for transitions, I think the tricky bit is that the headline associated with the current image probably should be at the top, so they need to be reordered, which is a different transition to the one we use in landscape.

### PR DESCRIPTION
**Pull request by triblondon**
See screenshot below.  I think this is fine to ship, but future work could include shuffling the stories so that the picture story headline is always at the top (a "rotating drum" type animation would work here)

image

### CODE REVIEW

**Comment by triblondon** [comment]
Also: I have not done anything with the time zone indicator as we're removing that anyway.

File: src/scss/mixins.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@mixin flex-container($orientation:row) {
```
Comment (JakeChampion) [inline src/scss/mixins.scss:1]: Nice idea :-) :+1:

**Comment by JakeChampion** [comment]
This currently has some conflicts, other than that LGTM

**Comment by triblondon** [comment]
@JakeChampion Ada will not be online ahead of the sprint review so I suggest we don't require a review from her otherwise we will not deliver this in the current sprint.

**Pull request merged**